src/Common.h

```diff
@@ -93,6 +93,12 @@ struct Core {
      */
     std::atomic<uint64_t>* localPinnedContexts;
 
+    /**
+     * Setting a jth bit indicates that the priority of the thread living at
+     * index j is temporarily raised.
+     */
+    std::atomic<uint64_t>* publicPriorityMask;
```

Comment:
This data structure has been around for a while; I can rename it in a future commit if we decide that that is a good idea. This is a pointer so that it can be allocated in its own cache line, since it is accessed from multiple threads and we want to reduce the risk of false sharing.

Reply:
Maybe better not to differentiate `public` vs `private` here; since most users will never see the private one.